### PR TITLE
added changelog and added info for version 0.2.0

### PR DESCRIPTION
**Pull request by coquelin77**
## Description

Implement a changelog to prepare for the release of version 0.2.0

Changes proposed:
- add CHANGELOG.md

## Type of change

New feature (non-breaking change which adds functionality)
Documentation update

Are all split configurations tested and accounted for?
- [x] yes
- [ ] no

Does this change require a documentation update outside of the changes proposed?
- [ ] yes
- [x] no

Does this change modify the behaviour of other functions?
- [ ] yes
- [x] no

Are there code practices which require justification?
- [ ] yes
- [x] no

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#v0.2.0
+
+This version varies greatly from the previous version (0.5.0). This version includes a great increase in
```
Comment (bhagemeier) [inline CHANGELOG.md]: Should be 0.1.0, which was the released version as of May 29th, 2019.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+"gpu" or "cpu". Make sure to specify the device when creating DNDarrays if the desired device is different than the
+defualt. If no device is specified then that device is assumed to be "cpu".
+
+####Basic Operations
+- [#308](https://github.com/helmholtz-analytics/heat/pull/308) balance
+- [#308](https://github.com/helmholtz-analytics/heat/pull/308) convert DNDarray to numpy NDarray
+- [#412](https://github.com/helmholtz-analytics/heat/pull/412) diag and diagonal
+- [#388](https://github.com/helmholtz-analytics/heat/pull/388) diff
+- [#362](https://github.com/helmholtz-analytics/heat/pull/362) distributed random numbers
+- [#327](https://github.com/helmholtz-analytics/heat/pull/327) exponenets and logarithms
```
Comment (bhagemeier) [inline CHANGELOG.md]: s/exponenets/exponents/

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- h5py >= 2.8.0
+- netCDF4 >= 1.4.0, <= 1.5.2
+- pre-commit >= 1.18.3 (development requirement)
+
+Additions
+=========
+##GPU Support
+[#415](https://github.com/helmholtz-analytics/heat/pull/415) GPU support was added for this release. To set the default device use `ht.use_device(dev)` where `dev` can be either
+"gpu" or "cpu". Make sure to specify the device when creating DNDarrays if the desired device is different than the
+defualt. If no device is specified then that device is assumed to be "cpu".
```
Comment (mtar) [inline CHANGELOG.md]: typo: defualt

Comment (bhagemeier) [reply CHANGELOG.md]: Well spotted ;)

**Pull request merged**